ppr-ui/src/store/store.ts

```diff
@@ -723,6 +723,7 @@ export const useStore = defineStore('assetsStore', () => {
 
   /** Actions **/
   function resetNewRegistration () {
+    state.value.registration.registrationNumber = null
```

Comment:
Call in dashboard on mount

![image](https://github.com/bcgov/ppr/assets/77707952/6ba25260-df5f-4fc3-a0f8-5a5bc1453064)